app/assets/stylesheets/splash.scss

```diff
@@ -0,0 +1,25 @@
+.splash_container {
+  // background-image: image-url('chalkboard.jpg');
+  // background-size: 100%;
+  // background-repeat: no-repeat;
+  background-color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.